README.md

```diff
@@ -31,7 +31,21 @@
 
 ## Introduction
 
-CredSweeper is a tool to detect credentials in any directories or files. CredSweeper could help users to detect unwanted exposure of credentials  (such as personal information, token, passwords, api keys and etc) in advance. By scanning lines, filtering, and using AI model as option, CredSweeper reports lines with possible credentials, where the line is, and expected type of the credential as a result.
+CredSweeper is a tool to detect credentials in any directories or files
+CredSweeper could help users to detect unwanted exposure of credentials
+(such as personal information, token, passwords, api keys etc.) in advance.
+By scanning lines, filtering, and using AI model as option,
+CredSweeper reports lines with possible credentials, where the line is,
+and expected type of the credential as a result.
+CredSweeper has experimental option ``--depth`` to scan files with predilection:
+- supported containers (tar, zip, gzip, bzip)
+- base64 encoded data
+- represent text (xml, html, json, yaml etc.) as structure
+- parse python source files with builtin ast engine
+
+Reported line numbers may be not actual in report but "info" field may help to understand
+how the credential was found.
```

Comment:
IMO introducing `--depth` option from here seems inappropriate..
`--depth` option should be introduced in other document.
Or you can create experimental section for experimental options or functions like this and write it there.

Reply:
The description was moved to guide.rst